module/Finna/src/Finna/View/Helper/Root/Record.php

```diff
@@ -1122,6 +1122,17 @@ public function getSourceIdElement()
         return $this->getLabelList();
     }
 
+    /**
+     * Render a list of record labels.
+     *
+     * @return string
+     */
+    public function getLabelList(): string
+    {
+        $this->driver->tryMethod('addResourceLabels');
```

Comment:
Miksi näin? Miksei vain yliajeta driverin `getLabels`-metodia palauttamaan kaikki tarpeellinen?